### PR TITLE
Feature create tables list #16

### LINKED ISSUE #16: Feature: Create tables list
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

image

image

**Describe alternatives you've considered**

**Additional context**
Web Frontend

### PR DESCRIPTION
**Pull request by Hubertgitck**
## Summary

Table list component for side panel

## Details

Created new component with angular material, added necessary imports.

## References

Used mat-expansion-panel https://material.angular.io/components/expansion/overview

image

### CODE REVIEW

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="tables-list">
+    <mat-accordion [togglePosition]="'before'">
+        <mat-expansion-panel>
+          <mat-expansion-panel-header>
+            <mat-panel-title class="tables-list__title">
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html]: BEM Element class should be on first child tag of BEM Block tag.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+              Tables
+            </mat-panel-title>
+          </mat-expansion-panel-header>
+          <div class="tables-list__items">
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html]: Same here.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.tables-list{
+    &__title{
+        font-weight: 500;
+    }
+
+    &__items p{
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.scss]: Do not use tags in styling, replace with classes.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-tables-list',
+  templateUrl: './tables-list.component.html',
+  styleUrls: ['./tables-list.component.scss'],
+})
+export class TablesListComponent {
+  panelOpenState = false;
```
Comment (StraykerPL) [inline binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.ts]: Are you using this value somewhere?

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Comment by StraykerPL** [comment]
Please add screenshot with visible component to description.

**Comment by StraykerPL** [comment]
Please update the branch.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html (hunk 1 above)
Comment (Hubertgitck) [reply binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html]: Refactored naming.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html (hunk 2 above)
Comment (Hubertgitck) [reply binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html]: Refactored naming.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.scss (hunk 3 above)
Comment (Hubertgitck) [reply binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.scss]: Replaced with class.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.ts (hunk 4 above)
Comment (Hubertgitck) [reply binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.ts]: No, value removed.

**Comment by StraykerPL** [comment]
Resolve conflicts.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html (hunk 1 above)
Comment (StraykerPL) [reply binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html]: This comment is not resolved.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html (hunk 2 above)
Comment (StraykerPL) [reply binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html]: This comment is not resolved.

File: binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html (hunk 1 above)
Comment (Hubertgitck) [reply binder-web-frontend/src/pages/home/components/tables-list/tables-list.component.html]: Refactored after hints from reviewer

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**